2024.html

```diff
@@ -61,6 +61,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://owen.blacker.me.uk/">Owen Blacker</a></li>
             <li><a href="https://boehs.org/">Evan Boehs</a></li>
             <li><a href="https://weblog.200ok.com.au/">Ben Buchanan</a></li>
+            <li><a href="https://benji.dog/">benji</a></li>
```

Comment:
Hello!

Can you move the line so that it appears by alphabetical order? (pseudo or family name)

Thanks!

Reply:
Sorry about that. Updated now!